redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  namespace: redirector-k8s-io-prod
+  labels:
+    app: redirector-k8s-io
+    version: dirty
```

Comment:
?

Reply:
Currently using master, once we agree on feasibility will switch to the latest stable release.